coreprofile/_index.md

```diff
@@ -1,9 +1,8 @@
 ---
 title: "Jakarta EE Core Profile"
-weight: -3
-summary: "The Jakarta EE Core Profile defines a profile of the Jakarta EE platform specifically targeted at smaller runtimes.
-"
+weight: -5
+summary: "The Jakarta EE Core Profile defines a profile of the Jakarta EE Platform targetting cloud native microservice applications."
```

Comment:
I am good with this rewording, but if I recall correctly there was some resistance to  the use of "cloud native"? But I may be mistaken...

Reply:
Changed